src/aleph_client/commands/pricing.py

```diff
@@ -211,9 +211,13 @@ def display_table_for(
                 if "vram" in tier:
                     row.append(f"{tier['vram'] / 1024:.0f}")
                 if "holding" in price_unit:
-                    row.append(
-                        f"{displayable_amount(Decimal(price_unit['holding']) * current_units, decimals=3)} tokens"
-                    )
+                    # If the pricing entity is confidential or compute units > 4, display "Not Available"
+                    if pricing_entity == PricingEntity.INSTANCE_CONFIDENTIAL or current_units > 4:
+                        row.append("Not Available")
+                    else:
+                        row.append(
+                            f"{displayable_amount(Decimal(price_unit['holding']) * current_units, decimals=3)} tokens"
+                        )
```

Comment:
The command for showing the pricing should allow to calculate and display it, only we need to block the instance creation, as there are some users that already have them and we have to show the pricing to them